src/index.tsx

```diff
@@ -228,35 +249,24 @@ export default function Dragger(props) {
     <div
       data-id='Dragger-outer'
       ref={outerEl}
-      className={`${styles.outer} ${isDraggingStyle ? styles.isDragging : ''}${props.disabled ? ' is-disabled' : ''} ${props.className}`}
+      className={`${styles.outer} ${isDraggingStyle ? styles.isDragging : ''}${props.disabled
+        ? ' is-disabled'
+        : ''} ${props.className}`}
       onTouchStart={onStart}
       onMouseDown={onStart}
-      style={{ ...props.style }}
-    >
+      style={{ ...props.style }}>
```

Comment:
Why spread the `props.style` and create a new object, you can just do `style={props.style}`

Reply:
I didn't write the personally.